docs/discover/field-statistics.asciidoc

```diff
@@ -0,0 +1,59 @@
+[[show-field-statistics]]
+== Explore the fields in your data
+
+beta::[]
+
+Explore the fields in your data with the *Field statistics* view in *Discover*.
```

Comment:
Since these are the Discover docs, do you need to say `in *Discover*`?

Reply:
What if you combined these two sentences:

`Explore the fields in your data with the *Fields statistics* view to answer questions such as:`